package.json

```diff
@@ -8,8 +8,9 @@
     "watch": "gulp watch",
     "lint": "eslint .",
     "pretest": "npm-shrinkwrap-check -3 && eslint .",
-    "test": "NODE_ENV=test mocha && npm run features",
-    "features": "NODE_ENV=test cucumber.js",
+    "test": "NODE_ENV=test mocha && NODE_ENV=test npm run features",
+    "db-migration": "knex migrate:latest",
```

Comment:
Should we name this "db-migrate" to keep the form consistent with the other scripts? ("features" is still the off one out though =/)